js/classes/entities/foreground/ui/hud.js

```diff
@@ -468,7 +472,9 @@ class PauseButton {
                     ASSET_MGR.resumeMusic();
                 }
             }
-        });
+        };
+        document.body.addEventListener("click", listener);
+        HUD.componentListeners.push([ "click", listener ]);
```

Comment:
So are we adding the listeners to a stack?

Reply:
It's technically an array, push is just a convenient way of adding elements. It's not really treated as a stack in the code aside from this line.